addons/cert-manager/metadata.yaml

```diff
@@ -14,3 +14,5 @@ deployTo:
 dependencies:
   - name: fluxcd
 
+needNamespace:
+  - vela-system
```

Comment:
If user set `parameter.tgtNs!="vela-system"`, then the addon install will fail again. The simplest way is to not let the user set namespace. Or better, you can take a look at `fluxcd` addon on how to handle custom namespaces.

Reply:
Good idear.